components/feature/downloads/src/main/res/values/strings.xml

```diff
@@ -45,4 +45,6 @@
     <string name="mozac_feature_downloads_third_party_app_chooser_dialog_title">Complete action using</string>
     <!-- Message that appears when trying to download with an external app fails. %1$s will be replaced with the name the external app. -->-->
     <string name="mozac_feature_downloads_unable_to_open_third_party_app">Unable to open %1$s</string>
+    <!-- Text for the info dialog when write to storage permissions have been denied but user tries to download a file. -->
+    <string name="mozac_feature_downloads_write_external_storage_permissions_needed_message">Storage access needed to download files. Go to Android settings, tap permissions, and tap allow.</string>
```

Comment:
nit: Should we mention or hint at which permission to allow? Tap allow is quite generic...users need to know it's the files and media permission.

Reply:
Ok maybe we could use "files and media permission access needed to download files. Go to Android settings, tap permissions, and tap allow."